packages/rocket-auth-aws-infrastructure/src/lambdas/challenge-answer.ts

```diff
@@ -0,0 +1,31 @@
+import { CognitoIdentityServiceProvider } from 'aws-sdk'
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda'
+import { errorResponse, okResponse } from './response'
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  try {
+    const params = JSON.parse(event.body!)
```

Comment:
Just curious, what's this '!' at the end of `event.body`??

Reply:
It's to "force" it

Here, `event.body` is nullable, so TS will complain about the fact that it is nullable and you need to write an `if` to make sure that it exists and react otherwise. If you write `event.body!`, you are telling TS to forget about the nullable part, so you "force" it into non-nullable.

The drawback of using that operator, and why I dislike its usage, is that it will `throw` if it is indeed `undefined`, but here it is okay to use it, because @charlietfe wrapped everything into a `try`/`catch`